typescript/commands/build.py

```diff
@@ -38,3 +40,4 @@ def compile_inferred_project(self, file_name, params=""):
             "cmd": cmd,
             "file_regex": "^(.+?)\\((\\d+),(\\d+)\\): (.+)$"
         })
+        self.build_parameters = params
```

Comment:
On second look, this might not be working, as you didn't pass the saved ``build_parameters`` to the ``compile_inferred_project`` callback. The ``params`` in the lambda expression means differently from the ``params`` elsewhere because they have different scope. Does this work on your machine?